runtime/bindings/python/tests/test_ngraph/test_node_factory.py

```diff
@@ -45,7 +46,49 @@ def test_node_factory_topk():
     node = factory.create(
         "TopK", arguments, {"axis": 1, "mode": "max", "sort": "value"}
     )
+    attributes = node.get_attributes()
 
     assert node.get_type_name() == "TopK"
     assert node.get_output_size() == 2
     assert list(node.get_output_shape(0)) == [2, 3]
+    assert attributes["axis"] == 1
+    assert attributes["mode"] == "max"
+    assert attributes["sort"] == "value"
+
+
+def test_node_factory_empty_topk():
+    factory = NodeFactory("opset1")
```

Comment:
@jiwaszki Why do we need to have NodeFactory in public API?

Reply:
We need to expose it so we can later use it to return Nodes. However it comes from `ngraph.utils.node_factory`, so we do not expose it as utils function.